holidays/locale/en_US/LC_MESSAGES/SI.po

```diff
@@ -25,18 +25,22 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "Generated-By: Lingua 4.15.0\n"
-"X-Generator: Poedit 3.2.2\n"
+"X-Generator: Poedit 3.5\n"
 
 #. New Year's Day.
 msgid "novo leto"
 msgstr "New Year's Day"
 
-#. Preseren's Day.
-msgid "Prešernov dan"
-msgstr "Preseren's Day"
+#. Prešeren's Day, the Slovenian Cultural Holiday.
+msgid "Prešernov dan, slovenski kulturni praznik"
+msgstr "Prešeren's Day, the Slovenian Cultural Holiday"
```

Comment:
As an alternative:
```suggestion
msgstr "Preseren's Day, the Slovenian Cultural Holiday"
```

Reply:
But [English Wiki](https://en.wikipedia.org/wiki/France_Pre%C5%A1eren) and [Slovenia's Government](https://www.gov.si/en/topics/national-holidays/) (in English version) use "š". 🙄

Reply:
I don't mind, just noticed it had been the only one with the "extended alphabet"

// missed Primož too